Sources/MapboxDirections/MBIntersection.swift

```diff
@@ -1,16 +1,14 @@
 import Foundation
-import CoreLocation
```

Comment:
SPM builds are failing because Swift for Linux doesn’t include Core Location. (Apparently penguin tracking isn’t a particularly commercially viable proposition.)

```
/Users/distiller/project/Sources/MapboxDirections/MBIntersection.swift:11:26: error: use of undeclared type 'CLLocationCoordinate2D'
    public let location: CLLocationCoordinate2D
                         ^~~~~~~~~~~~~~~~~~~~~~
```